libsolidity/codegen/CompilerContext.cpp

```diff
@@ -331,6 +344,12 @@ bool CompilerContext::appendCallback(evmasm::AssemblyItem const& _i) {
 						mstore(add(callBytes, 0x24), offset)
 						mstore(add(callBytes, 0x44), length)
 						kall(callBytes, 0x64, destOffset, length)
+						
```

Comment:
We actually weren't even doing the zero-ing out here before, so I went ahead and put in the correct version.  Looked over all the other opcodes and I think this is the only non-call-type which is affected.